src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -411,6 +411,17 @@ protected override int MakeSlot(BoundExpression node)
         {
             switch (node.Kind)
             {
+                case BoundKind.ThisReference:
+                case BoundKind.BaseReference:
+                    {
+                        var method = _member as MethodSymbol;
```

Comment:
>_member [](start = 37, length = 7)

The name of this field implies it is a member, i.e. a top level method when it is a method. Is this not the case?

Reply:
It is not necessarily a type member. Renamed.

---
In reply to: [241169548](https://github.com/dotnet/roslyn/pull/31703#discussion_r241169548) [](ancestors = 241169548)